### PR TITLE
GAed and redesigned MongoDB dashboards 

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

**Review by YashSartanpara1 — APPROVED** [review]

File: docs/_images/MongoDB_ReplSetSummary_New.png
Comment (sleto-it) [inline docs/_images/MongoDB_ReplSetSummary_New.png:1]: Low priority feedback (we can implement also later if we have a todo open somewhere). Would be nice to have an image that: do not have that "Experimental" in the title.  And also that "_(New)" as I think (hope) we removed it. Same for the image for the Sharded Clsuter Summary

File: docs/details/dashboards/dashboard-mongodb-cluster-summary.md
Comment (sleto-it) [inline docs/details/dashboards/dashboard-mongodb-cluster-summary.md:1]: I suggest we add a `-old` in the file name as well.  If we do so, we can use the current file name for the new dashboard.  So things are transparent. The new dashboard is a replacement of the old.  The links will continue to work

File: docs/details/dashboards/dashboard-mongodb-replset-summary.md
Comment (sleto-it) [inline docs/details/dashboards/dashboard-mongodb-replset-summary.md:1]: same here - I suggest we add a `-old` in the file name.

File: docs/details/dashboards/dashboard-replsetsummary-new.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+
+# MongoDB ReplSet Summary (NEW)
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-replsetsummary-new.md:2]: Consider to remove "(NEW)". It is not included in the dashboard name (in the product)

File: docs/details/dashboards/dashboard-replsetsummary-new.md
Comment (sleto-it) [inline docs/details/dashboards/dashboard-replsetsummary-new.md:1]: Suggest to use old file name.

File: docs/details/dashboards/dashboard-replsetsummary-new.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+### Command Operations
+
+Shows the rates of different MongoDB operations per second, including primary operations (like queries, inserts, updates, and deletes), replicated operations on secondary nodes, and automatic deletions by TTL indexes. 
+
+It helps you visualize your database's workload, showing how different types of operations contribute to overall activity. Use this to spot unusual patterns, balance between read and write operations, and understand your MongoDB instance's performance at a glance.
+
+### Top Hottest Collections by Read
+
+Lists the five collections with the highest read activity. Use this The panel to quickly identify which collections are under the most demand, allowing you to monitor read-heavy workloads and optimize performance accordingly.
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-replsetsummary-new.md:78]:
```suggestion
Lists the five collections with the highest read activity. Use this panel to quickly identify which collections are under the most demand, allowing you to monitor read-heavy workloads and optimize performance accordingly.
```

File: docs/details/dashboards/dashboard-replsetsummary-new.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+### Query Execution Times
+
+Displays the average latency of operations, categorized by read, write, or command. It visualizes how long each type of operation takes to execute over time, helping you identify trends or potential performance bottlenecks in your database operations Use this panel to you to monitor and optimize the responsiveness of your MongoDB cluster.
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-replsetsummary-new.md:82]:
```suggestion
Displays the average latency of operations, categorized by read, write, or command. It visualizes how long each type of operation takes to execute over time, helping you identify trends or potential performance bottlenecks in your database operations. Use this panel to you to monitor and optimize the responsiveness of your MongoDB cluster.
```

File: docs/details/dashboards/dashboard-replsetsummary-new.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+### Queued Operations
+
+Displays the number of operations that are queued due to locks within your MongoDB cluster. It helps identify performance bottlenecks by showing how many operations are waiting because of locking issues. 
+
+Use this panel to track these queued operations and monitor the impact of locking on system performance over time and take action if necessary.
+
+### Reads & Writes
+
+Tracks the number of read and write operations over time in your MongoDB environment. Reads represent data queries, while writes represent data modifications. 
+
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-replsetsummary-new.md:103]:
```suggestion
```

File: docs/details/dashboards/dashboard-replsetsummary-new.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+Use this panel to get insights into the workload distribution and monitor the performance of database operations, ensuring that the system is handling read and write operations efficiently.
+
+### Connections
+
+monitors the average number of active and available MongoDB connections over time.
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-replsetsummary-new.md:109]:
```suggestion
Monitors the average number of active and available MongoDB connections over time.
```

File: docs/details/dashboards/dashboard-sharded-cluster-summary.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
+# Sharded Cluster Summary
+
+This dashboard provides a comprehensive view of your MongoDB sharded cluster's performance, health, and resource utilization. It offers insights into shard distribution, query operations, data balance, and replication status across your entire cluster.
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-sharded-cluster-summary.md:3]:
```suggestion
This dashboard provides a comprehensive view of your MongoDB sharded cluster's performance, health, and resource utilization. It offers insights into shard distribution, query operations, data balance, and replication status across your entire cluster.

For MongoS (Router) specific monitoring, see the [MongoDB Router Summary](dashboard-mongodb-router-summary.md) dashboard.
```

File: docs/details/dashboards/dashboard-sharded-cluster-summary.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+### QPS of Services in Shard
+ 
+Displays the Queries Per Second (QPS) for each shard and the config server replica set in your MongoDB cluster. It shows the rate of operations (excluding commands) for each component, helping you quickly assess the query load distribution across your sharded environment. 
+
+The chart uses the most recent non-null value and updates based on your selected time interval. This visualization allows you to easily identify which shards are handling the most queries and spot any potential load imbalances. 
+
+### Shards
+
+A shard contains a subset of sharded data for a sharded cluster. Together, the cluster’s shards hold the entire data set for the cluster.
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-sharded-cluster-summary.md:17]: I think we should add what we are reporting, in this case, the number of shards (as we do below for mongos) rather what a shard is

Comment (sleto-it) [inline docs/details/dashboards/dashboard-sharded-cluster-summary.md:17]:
```suggestion
Number of shards of this cluster. A shard contains a subset of sharded data for a sharded cluster. Together, the cluster’s shards hold the entire data set for the cluster.
```

File: docs/details/dashboards/dashboard-sharded-cluster-summary.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+### Mongos
+
+Number of mongos routers registered as part of the cluster.
+
+### Draining Shards
+
+When you run removeShard, MongoDB drains the shard by using the balancer to move the shard’s chunks to other shards in the cluster. Once the shard is drained, MongoDB removes the shard from the cluster.
```
Comment (sleto-it) [inline docs/details/dashboards/dashboard-sharded-cluster-summary.md:25]: As we report a number, again I think here we have to mention what is this number (in addition to any other other explanations). Like we did above, we reported what the number is, number of shards, number of mongos, etc

**Review by sleto-it — APPROVED** [review]
Added some more comments. I think after those, OK to merge, and we will anyway continue the review in the wider `2.43.0` PR, this time with preview available Some images might be improved, I believe, but can be done later.  The column `Elements` in https://docs.percona.com/percona-monitoring-and-management/details/dashboards/index.html might be updated (or removed). I do not fully get it, and seems hard to maintain.  Approving for now.

**Pull request merged**